internal/hud/view/model_to_view.go

```diff
@@ -0,0 +1,43 @@
+package view
+
+import (
+	"time"
+
+	hudmodel "github.com/windmilleng/tilt/internal/hud/model"
+	tiltmodel "github.com/windmilleng/tilt/internal/model"
+)
+
+func NewView(m hudmodel.Model) View {
+	var resources []Resource = nil
+	for name := range m.Resources {
```

Comment:
can do `for name, r := range...` and then don't need to do `m.Resources[name]` later